cli/tsc/99_main_compiler.js

```diff
@@ -1213,6 +1213,46 @@ delete Object.prototype.__proto__;
 
   ts.deno.setNodeBuiltInModuleNames(nodeBuiltInModuleNames);
 
+  // list of globals that should be kept in Node's globalThis
+  ts.deno.setNodeOnlyGlobalNames([
+    // when bumping the @types/node version we should check if
+    // anything needs to be updated here
```

Comment:
I moved this from the fork to here: https://github.com/denoland/TypeScript/pull/4#discussion_r1139390659

The reason is because it's dependent on the `@types/node` version and it slowly makes the fork less node specific.

Reply:
Makes sense, thanks for explanation